lucien-ios/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UIViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/4/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+extension UIViewController {
+
+}
+
+extension UILabel {
```

Comment:
Since this is a `UILabel` extension, it should live in a file called `UILabel+Extensions`, not here